realm/realm-library/src/main/java/io/realm/RealmQuery.java

```diff
@@ -1092,6 +1093,29 @@ public boolean isValid() {
         return this;
     }
 
+    /**
+     * Returns a distinct set of objects of a specific class. As a Realm is unordered, it is
+     * undefined which objects are returned in case of multiple occurrences.
```

Comment:
@beeender but the original order is undefined unless you sorted it, so I think this is accurate? Perhaps adding "unless sorted, in which case the first object is returned."